codingstandards/continuousintegration.md

```diff
@@ -0,0 +1,55 @@
+# Database Continuous Integration
+## Overview
+Database Continuous Integration (CI) is a method in which changes to the database are recorded incrementally in order to provide a continuously updated database. This form of integration allows for developers to have local copies of a database and to update their local database incrementally.  
+
+## Install vs. Update
+For each database using the CI design there are two levels of versioning that need to be changed. They are `install` and `update`.
+
+### Install
+The `install` changes are all the changes needed to create the database from scratch. This will include all table declorations and stored procedures. The `install` changes will be applied whenever a devoloper wants to create a database. This reqires the `install` directory to have **the current state of the database**. All changes to the database will modify the `install` directory.
+
+### Update
+The `update` changes are all the changes needed to get the database from point **A** to point **B**. The `update` changes are versioned with a unique modifier so that the database knows what updates it has completed and which ones it needs to run. The PROD database will run the `update` changes whenever a new change is released. 
```

Comment:
How about something like "...get existing databases from point **A** to point **B**."

Also, the last sentence, be more clear that it isn't just PROD: "An existing environment, such as TEST or PROD, will have the `update` changes applied when the new version is released."